playlist_collection/m3u_to_playlist.py

```diff
@@ -41,7 +50,7 @@ def m3u_to_playlist(ssn, library_name: str, file_path: str, users: list=['@me'])
 	for lib in sections:
 		if lib['title'] == library_name:
 			for user_token in user_data.values():
-				ssn.post(f'{base_url}/playlists/upload', params={'sectionID': lib['key'], 'path': file_path, 'X-Plex-Token': user_token})
+				ssn.post(f'{base_url}/playlists/upload', params={'sectionID': lib['key'], 'path': library_path+file_path, 'X-Plex-Token': user_token})
```

Comment:
Using `+` is a bit dangerous to do. The filepath is for a file on the _server_ so this should be in the _servers os_ format. If you're running Linux but the server is on Windows, then you need to supply a windows path. Using `os.path.join` won't work either as that uses the delimiter of the client, not the server. Best we can do is make it extra clear in the documentation (in argparse and at the top of the file) that the path supplied needs to be in the format of the servers OS and that it should end with that delimiter.